src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -392,6 +409,12 @@ public void NormalizePathEmpty()
         [Fact]
         public void NormalizePathBadUNC1()
         {
+            // UNC is not useful outside Windows
+            if (!NativeMethodsShared.IsWindows)
```

Comment:
Do we want non-Windows to run this and report success? Or have a trait and skip tests on certain platforms?